components/01-atoms/DappHeader.tsx

```diff
@@ -35,6 +37,10 @@ export const DappHeader = () => {
   const router = useRouter();
   const currentRoute = router.pathname;
 
+  const isConnectedToTestnet = () => {
+    return !!chain && chain.id === sepolia.id;
+  };
```

Comment:
```suggestion
  const isConnectedToTestnet = () => {
    return !!chain && chain.id !== mainnet.id;
  };
```

Reply:
@FrancoAguzzi  I disagree with this one. We only want to display the banner when the user is connected to a specific testnet network, not on every network except mainnet.

Reply:
I get it. We can align with the team how this will be, I imagine we might want to show them they are not registering the name in Ethereum Mainnet. Let's discuss with the ENS squad.

P.S. Please do not resolve threads where the conversation has not come to an agreement